src/client/Inode.cc

```diff
@@ -27,6 +28,12 @@ Inode::~Inode()
     assert(oset.objects.empty());
   }
 
+  if (!delegations.empty()) {
+    lsubdout(client->cct, client, 0) << __func__ << ": leftover delegations on inode 0x"
+      << std::hex << ino << std::dec << dendl;
+    assert(delegations.empty());
```

Comment:
@jtlayton in place of assert(expression), We can place exit() here.
it will save evaluation of expression. Since we always want to exit here.

Reply:
Sure...